src/eduid/satosa/scimapi/statsd.py

```diff
@@ -0,0 +1,56 @@
+import logging
+import re
+from typing import Any, Mapping
+
+import satosa.internal
+from satosa.micro_services.base import ResponseMicroService
+
+from eduid.common.config.base import StatsConfigMixin
+from eduid.common.stats import init_app_stats
+
+logger = logging.getLogger(__name__)
+
+
+class requester_counter(ResponseMicroService):
```

Comment:
Should be renamed to RequesterCounter as python classes should be CamelCase

Reply:
I changed this from camel to snake because SonarCloud complained. Who to trust?

Reply:
Even if SonarCloud stated that "The style guide PEP-8 [...] but also accepts the "snake_case" convention" I miss read the issue. Will change it back!